css/index.css

```diff
@@ -0,0 +1,3 @@
+body {
+	background-color: blue;
```

Comment:
I assume you wrote this to test your CSS was linked properly. Don't push things like this to master - consider this your finished product.